scss/_variables-dark.scss

```diff
@@ -6,46 +6,81 @@
 // Global colors
 //
 
+// Boosted mod: redefine basic colors
+// scss-docs-start brand-colors-dark
+$functional-green-dark:  #32c832 !default;
+$functional-blue-dark:   #527edb !default;
+$functional-yellow-dark: $functional-yellow !default;
+$functional-red-dark:    #d53f15 !default;
+// scss-docs-end brand-colors-dark
+// End mod
+
 // scss-docs-start sass-dark-mode-vars
+// scss-docs-start theme-color-dark-variables
+$primary-dark:                      $orange !default;
+$secondary-dark:                    $black !default;
+$success-dark:                      $functional-green-dark !default;
```

Comment:
Don't we need to override `$blue`, `$green`, etc. + `$colors` or create `*-dark` versions of them?

In light mode we have:

```scss
$functional-green:  #228722 !default;
$green: $functional-green !default;
$success: $green !default;
```

In dark mode:

```scss
$functional-green-dark:  #228722 !default;
$success: $functional-green-dark !default;
```